tsconfig.json

```diff
@@ -11,7 +11,7 @@
     "target": "es2015",
     "module": "esnext",
     "typeRoots": ["node_modules/@types"],
-    "lib": ["es2017", "dom"],
+    "lib": ["es2017", "es2019", "dom"],
```

Comment:
I think this shouldn't be global because It is undesirable to use es2019 on the fronted side since not all browsers support new standards. Put it in tsconfig.lib.json inside the specific library.